cli/src/main/java/com/devonfw/tools/ide/commandlet/CreateCommandlet.java

```diff
@@ -63,6 +63,7 @@ public void run() {
     this.context.setIdeHome(newProjectPath);
     this.context.verifyIdeMinVersion(true);
     super.run();
+    this.context.verifyIdeMinVersion(true);
```

Comment:
this will now happen after all the software is installed, etc.
It would be an option to do it in `reloadContext` method.
To override it, it would also need to be changed from `private` to `protected`. 
I am happy to merge as is for our release today.
We should create a PR for further improvements.
Maybe we even find a better way or directly integrated it into the super-class what IMHO makes the most sense:
If I do `ide update` and the `settings` get updated and `IDE_MIN_VERSION` is changed to a newer value then it should also fail if I am outdated.

Reply:
I created issue #1311 for this.